src/components/Header/header.scss

```
@@ -16,17 +20,36 @@
     }
 
     &__title {
-        font-weight: bolder;
+        font-weight: 1000;
         font-size: 2.5rem;
-        background: linear-gradient(135deg, $red, $orange);
+        text-transform: uppercase;
+        color: transparent;
+        background-image: linear-gradient(145deg, $red, lighten($orange, 5%));
+        background-size: 400%;
         -webkit-background-clip: text;
         background-clip: text;
-        -webkit-text-fill-color: transparent;
+        animation: moving-title-gradient 20s infinite alternate;
     }
 
     &__subtitle {
         font-weight: 300;
         color: darken($color: $orange-grey, $amount: 30%);
         padding: 1rem;
     }
+
+    &__children {
+        position: relative;
+        width: 100%;
+        text-align: center;
```

Talvez deixar o `header__children` na esquerda
```suggestion
        margin-left: 1rem;
```